README.md

```diff
@@ -39,13 +42,23 @@ You can find human-readable API documentation generated from the YAML file here:
 
 ## Adoption
 
-We are currently working with pinning services to expose this API — so stay tuned!  ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
-  - `{your project could go here}`
-
 Built-in support for pinning services exposing this API is coming to IPFS tooling in Q3: 
   - [go-ipfs](https://github.com/ipfs/go-ipfs) / [js-ipfs](https://github.com/ipfs/js-ipfs) (CLI/HTTP API)  ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
   - [ipfs-desktop](https://github.com/ipfs-shipyard/ipfs-desktop) / [ipfs-webui](https://github.com/ipfs-shipyard/ipfs-webui) (GUIs) ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
 
+Below is the growing list of libraries and services implementing the API.
```

Comment:
I would just remove this line. Looking at this in preview I just read right over this because it blends in with the body text above it. I think the headers below are descriptive enough.